lib/globus_client.rb

```diff
@@ -22,8 +22,10 @@ class << self
     # @param transfer_endpoint_id [String] the transfer API endpoint ID supplied by Globus
     # @param transfer_url [String] the transfer API URL
     # @param auth_url [String] the authentication API URL
-    def configure(client_id:, client_secret:, uploads_directory:, transfer_endpoint_id:, transfer_url: default_transfer_url, auth_url: default_auth_url)
-      instance.config = OpenStruct.new(
+    # rubocop:disable Metrics/ParameterLists
+    def configure(client_id:, client_secret:, uploads_directory:, transfer_endpoint_id:,
+                  transfer_url: default_transfer_url, auth_url: default_auth_url)
+      instance.config = OpenStruct.new( # rubocop:disable Style/OpenStructUse
```

Comment:
I wonder if we might want to consider using a `Struct` instance with keyword arguments instead of `OpenStruct`. It's probably a defensible change, though I'm not too worried about performance in this context.

Reply:
I'm more than happy to follow up with any refactors like that in another PR.